lib/mocha/class_method.rb

```diff
@@ -96,12 +96,11 @@ def to_s
     end
 
     def method_visibility
-      symbol = method_name.to_sym
       metaclass = original_method_owner
 
-      (metaclass.public_method_defined?(symbol) && :public) ||
-        (metaclass.protected_method_defined?(symbol) && :protected) ||
-        (metaclass.private_method_defined?(symbol) && :private)
+      (metaclass.public_method_defined?(method_name) && :public) ||
+        (metaclass.protected_method_defined?(method_name) && :protected) ||
+        (metaclass.private_method_defined?(method_name) && :private)
```

Comment:
> The `xxx_method_defined?` methods accept either a symbol or string so we don't need to manually convert the method name to a symbol.

Is this definitely true in all versions of Ruby? I have a vague recollection that this may have changed over time.